bot/Makefile

```diff
@@ -1,28 +1,38 @@
 run-rocketchat:
 	make config-bot
+	make download-model
```

Comment:
`make download-model` é executado inúmeras vezes, é possível fazer esse comando ser executado de forma automática? um [exemplo pre-build](https://stackoverflow.com/questions/1612278/pre-build-step-in-makefile)

Reply:
Tendo em vista a re-analise do código, não se viu necessidade de fazer o que foi pedido.